engine/src/flutter/shell/platform/darwin/ios/framework/Headers/FlutterPlugin.h

```diff
@@ -425,6 +425,27 @@ typedef enum {
 - (nullable NSObject*)valuePublishedByPlugin:(NSString*)pluginKey;
 @end
 
+#pragma mark -
+/**
+ * The target of registration of plugins.
+ *
+ * This often is hooked up to the GeneratedPluginRegistrant which is
+ * automatically generated by Flutter for the dependencies listed in the
+ * project.
+ */
+@protocol FlutterPluginRegistrant <NSObject>
```

Comment:
This may be confused with `GenertedPluginRegistrant`. Just from the naming, one would think `GenertedPluginRegistrant` adopts `FlutterPluginRegistrant`, but it is not the case. 

But I can't think of a better name.